include/dlaf/gpu/error.h

```diff
@@ -15,26 +15,28 @@
 #include <exception>
 #include <iostream>
 
-#ifdef DLAF_WITH_CUDA
-#include <cuda_runtime.h>
-#endif
-
 #include "dlaf/common/source_location.h"
+#include "dlaf/gpu/api.h"
 
-namespace dlaf {
-namespace internal {
+#ifdef DLAF_WITH_GPU
 
-#ifdef DLAF_WITH_CUDA
+namespace dlaf::gpu {
```

Comment:
Is the move to a non-internal namespace intentional?

Reply:
Yes, but checking again, maybe is not a good idea as the checkError functions should be called through macros...

Reply:
I'll update this today.

Reply:
This now internal, closing.